CONTRIBUTING.adoc

```diff
@@ -43,7 +43,7 @@ You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker in
 Here's how to get set up:
 
 1. For Go, Git and optionally also Docker, follow the links below to get to installation information for these tools: +
-** http://golang.org/doc/install[Installing Go]. You must install Go 1.4 and NOT use $HOME/go directory for Go installation.
+** http://golang.org/doc/install[Installing Go]. Currently, OpenShift supports building using Go 1.6.x.
```

Comment:
@smarterclayton do you remember what the issue with $HOME/go was, and whether it still applies?